README.md

```diff
@@ -1,2 +1,23 @@
 # skycoin-services
+
 skycon-services
+
+## dmsg daemon
+
+This daemon will create `.dmsg-uuid` file, which contains your unique UUID.  
```

Comment:
Not sure if it was part of the requirements you got, but putting logs and config/ID values in the same file may be less than optimal. Unless this was requested specifically.